libs/designer/src/lib/core/state/mcp/panel/mcpPanelSlice.ts

```diff
@@ -15,18 +15,20 @@ export interface PanelState {
   currentPanelView?: ConfigPanelView;
   selectedTabId?: string;
   selectedConnectorId: string | undefined;
+  selectedOperationId: string | undefined;
```

Comment:
Lets not store operationId in the store.... but if that is causing lot of changes right now, you can include that in next PR